examples/README.md

```diff
@@ -246,49 +246,49 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>SSD ResNet50 V1</td>
     <td>Object Detection</td>
     <td>Post-Training Static Quantization</td>
-    <td><a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/object_detection/tensorflow_models/quantization/ptq">pb</a> / <a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/object_detection/tensorflow_models/quantization/ptq">ckpt</a></td>
+    <td><a href="./examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq">pb</a> / <a href="./examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq">ckpt</a></td>
```

Comment:
Please fix the hyperlink issue.

Reply:
And also remove the `*` before new API examples. 
![image](https://user-images.githubusercontent.com/51692656/215409762-d9c7b471-d4ae-43bc-9430-9b361a492b45.png)